packages/vhd-lib/src/Vhd/VhdAbstract.integ.spec.js

```diff
@@ -0,0 +1,140 @@
+/* eslint-env jest */
+
+import rimraf from 'rimraf'
+import tmp from 'tmp'
+import fs from 'fs-extra'
+import { getSyncedHandler } from '@xen-orchestra/fs'
+import { Disposable, pFromCallback } from 'promise-toolbox'
+
+import { openVhd } from '../index'
+import { createRandomFile, convertFromRawToVhd, createRandomVhdDirectory } from '../tests/utils'
+import { VhdAbstract } from './VhdAbstract'
+
+let tempDir
+
+jest.setTimeout(60000)
+
+beforeEach(async () => {
+  tempDir = await pFromCallback(cb => tmp.dir(cb))
```

Comment:
You could make this the root of the remote.

Reply:
As I said https://github.com/vatesfr/xen-orchestra/pull/5962#discussion_r743505223 , I will need multiple remotes in test